src/Provider/BlizzardAPAC.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/*!
+* Hybridauth
+* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
+*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
+*/
+
+namespace Hybridauth\Provider;
+
+use Hybridauth\Adapter\OAuth2;
+use Hybridauth\Exception\UnexpectedApiResponseException;
+use Hybridauth\Data;
+use Hybridauth\User;
+
+/**
+ * Blizzard US/SEA Battle.net OAuth2 provider adapter.
+ */
+class BlizzardAPAC extends OAuth2
```

Comment:
just extend `Blizzard` and change this three properties:
```
/**
 * {@inheritdoc}
 */
protected $apiBaseUrl = 'https://apac.api.battle.net/';

/**
 * {@inheritdoc}
 */
protected $authorizeUrl = 'https://apac.battle.net/oauth/authorize';

/**
 * {@inheritdoc}
 */
protected $accessTokenUrl = 'https://apac.battle.net/oauth/token';
```

Reply:
Ahh. That would be a better approach. I'll do that later today if I get the chance.

Reply:
Oops! My bad! I completely forgot about this, I'll do it when I get some time with a computer. I've been a bit busy recently.